game_patch/multi/server.cpp

```diff
@@ -96,6 +96,52 @@ void load_additional_server_config(rf::Parser& parser)
         }
     }
 
+    if (parser.parse_optional("$DF Weapon Stay Exemptions:")) {
```

Comment:
> Nitpick. But I think exclude or exclusions is better.

I don't like "exemptions" too. I think it's not a popular word. Excludes/exclusions sounds more familiar. 
Anyway it would make sense to include Fusion Launcher on that list by default and allow changing it to an empty list or whatever the server owner wants. Like if we are making it configurable, there is no reason to have Fusion as a special case.